src/neptune/types/series/float_series.py

```diff
@@ -96,3 +100,15 @@ def unit(self):
 
     def __str__(self):
         return "FloatSeries({})".format(str(self.values))
+
+    def clear_of_unsupported_values(self):
+        cleared_values = []
+        for value in self._values:
+            if is_unsupported_float(value):
+                warn_once(
+                    message=f"WARNING: The value you're trying to log (`{str(value)}`) is a non-standard float "
+                    f"value that is not currently supported."
```

Comment:
```suggestion
                    message=f"WARNING: A value you're trying to log (`{str(value)}`) will be skipped because "
                    f"it's a non-standard float value that is not currently supported."
```

Reply:
Changed :D